jest-setup.js

```diff
@@ -1,5 +1,8 @@
 import 'whatwg-fetch';
 import 'react-native-gesture-handler/jestSetup';
+import mockAsyncStorage from '@react-native-async-storage/async-storage/jest/async-storage-mock';
+
+jest.mock('@react-native-async-storage/async-storage', () => mockAsyncStorage);
```

Comment:
what if not doing this?

Reply:
the tests fail because the async storage is not properly defined for jest, if we do not mock it.